quint/test/ir/IRFinder.test.ts

```diff
@@ -64,12 +64,15 @@ describe('findParameterWithId', () => {
   const modules = [buildModuleWithDecls(['pure def x(a: int): int = a'])]
 
   it('finds definition for existing id', () => {
-    const def = findParameterWithId(modules, 1n)
-
+    const def = findParameterWithId(modules, 2n)
```

Comment:
The number is shifted because the type annotation in the param is now the first leaf we hit while parsing this example.